### DISCUSSION TITLE
My star page is taking too long to load

### DISCUSSION THREAD

**Discussion by scott306lr** [community/community#23472] [New to GitHub]
<p>I cannot access my star page when I’m logged in, tested on other browsers and the same issue happens.<br>
This problem had been there for a few weeks.</p>

**Comment by jfullstackdev** [comment]
<aside class="quote no-group" data-username="scott306lr" data-post="1" data-topic="252174">
<div class="title">
<div class="quote-controls"></div>
 scott306lr:</div>
<blockquote>
<p>I cannot access my star page</p>
</blockquote>
</aside>
<p>what star page is that ?</p>
<p>the Stars tab ?</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/e/f/ef85c6f9a470b40680ccf9e31f7f2b3837319fee.png" data-download-href="/uploads/short-url/yaUJniiPGElYmgNEz8ZGTdYxnQy.png?dl=1" title="image"><img src="https://user-images.githubusercontent.com/47092464/181077264-f35ffd60-3739-4018-b973-5d3aea43691a.png" alt="image" data-base62-sha1="yaUJniiPGElYmgNEz8ZGTdYxnQy" width="690" height="134" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/e/f/ef85c6f9a470b40680ccf9e31f7f2b3837319fee_2_690x134.png, https://aws1.discourse-cdn.com/github/original/3X/e/f/ef85c6f9a470b40680ccf9e31f7f2b3837319fee.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/e/f/ef85c6f9a470b40680ccf9e31f7f2b3837319fee.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/e/f/ef85c6f9a470b40680ccf9e31f7f2b3837319fee_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">898×175 48 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>
<p>try logging in Incognito mode too</p>

**Comment by scott306lr** [comment]
<p>Yes, the Stars tab. The error message “This page is taking too long to load.” will jump out.<br>
I can visit my stars page in Incognito mode, but when I’m logged in, the error appear again.<br>
I guess this issue appears after I renamed one of the lists, not certain in this.<br>
Is there possibly be a way to fix this issue?</p>

**Comment by jfullstackdev** [comment, marked as answer]
<p>try to delete the entire history of the regular tab because it’s working in Incognito</p>

**Comment by scott306lr** [comment]
<p>It worked, but it still takes for nearly 10 seconds loading, while Incognito takes for like 2.<br>
Thanks a lot still!</p>
